src/main/java/io/stargate/sgv2/jsonapi/JsonapiRetriableStargateBridge.java

```diff
@@ -0,0 +1,21 @@
+package io.stargate.sgv2.jsonapi;
+
+import io.quarkus.grpc.GrpcService;
+import io.stargate.sgv2.api.common.grpc.qualifier.Retriable;
+import jakarta.annotation.Priority;
+import jakarta.enterprise.inject.Alternative;
+import jakarta.inject.Singleton;
+import jakarta.ws.rs.Produces;
+
+@Alternative
+@Priority(1)
+@GrpcService
+@Singleton
+public class JsonapiRetriableStargateBridge {
```

Comment:
Nit: this should probably be named "...BridgeProvider" since it returns bogus (`null`) instance but isn't one.